src/integrationTest/java/uk/gov/hmcts/reform/amapi/exception/AccessManagementResponseEntityExceptionHandlerTest.java

```diff
@@ -76,7 +76,7 @@ public void testHandleHttpMessageNotReadable() throws Exception {
         String invalidJson = Resources.toString(Resources
             .getResource("exception-mapper-data/malformedInput.json"), StandardCharsets.UTF_8);
 
-        this.mockMvc.perform(post("/api/filter-resource")
+        this.mockMvc.perform(post("/api/v1/filter-resource")
```

Comment:
Please define a global "version" variable and replace all of the version hardcoding in this file

Reply:
done